tomcat/src/main/java/org/apache/coyote/http11/RequestMapping.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.coyote.http11;
+
+import org.apache.coyote.http11.controller.*;
+import org.apache.coyote.http11.request.HttpRequest;
+
+import java.util.List;
+
+public class RequestMapping {
+    private static final List<Controller> CONTROLLER_INSTANCES = List.of(
+            new LoginGetController(), new LoginPostController(), new RegisterGetController(), new RegisterPostController(), new DefaultGetController()
```

Comment:
```suggestion
            new LoginGetController(),
            new LoginPostController(), 
            new RegisterGetController(), 
            new RegisterPostController(), 
            new DefaultGetController()
```

한 줄에 길게 늘어뜨려 쓰지 말 것
엔터로 의미를 구분할 것